main/guides/getting-started/how-to-use-ui-kit.md

```diff
@@ -0,0 +1,142 @@
+# Introduction to Agoric UI Kit
+The Agoric UI Kit is a collection of components and tools designed to help developers build graphical user interfaces for Agoric decentralized applications (dapps). It provides a set of reusable React components, utilities for reading contract data from vstorage, functions for connecting to the user's wallet, and methods for executing offers on the Agoric blockchain.
+
+## Setup
+To use the Agoric UI Kit, you need to set up SES in your application environment. SES is a runtime for JavaScript that ensures the security and integrity of your Agoric dapp. Refer to the [SES setup example](https://github.com/Agoric/dapp-inter/blob/main/src/main.tsx#L1) for instructions on enabling SES in your application. 
+
+## React Components
+The UI Kit provides a set of React components in the `packages/react-components` directory. These components are designed to make it easier to build user interfaces for Agoric dapps using React. To use these components, simply import them into your React application and integrate them into your UI.
+
+## Reading Contract Data (vstorage)
+The UI Kit includes utilities for reading data from vstorage, the virtual storage system used by Agoric contracts. The `makeAgoricChainStorageWatcher` function allows you to subscribe to updates to vstorage and efficiently retrieve data using the Agoric RPC API.
+
+```js
+import {
+  makeAgoricChainStorageWatcher,
+  AgoricChainStoragePathKind as Kind,
+} from '@agoric/rpc';
+
+const watcher = makeAgoricChainStorageWatcher(restApi, chainName);
+
+// Watch vstorage children at a given node.
+const stopWatching = watcher.watchLatest<string[]>(
+  [Kind.Children, 'published.vaultFactory.managers'],
+  managerIds => {
+    console.log('Got vault manager IDs:', managerIds);
+  },
+);
+
+// Stop watching.
+stopWatching();
+
+// Watch vstorage data at a given node.
+watcher.watchLatest<Brands>(
+  [Kind.Data, 'published.agoricNames.brand'],
+  brands => {
+    console.log('Do something with the brands');
+  },
+);
+```
+
+### Connecting to User's Account (Keplr)
```

Comment:
Worth pointing out if you're using react, you can use @agoric/react-components instead of this API.

Reply:
Added that note to the explainer